generators/client-2/index.js

```diff
@@ -279,37 +279,8 @@ module.exports = JhipsterClientGenerator.extend({
     writing: writeFiles(),
 
     install: function () {
-        var injectDependenciesAndConstants = function () {
-            if (this.options['skip-install']) {
-                this.log(
-                    'After running ' + chalk.yellow.bold('npm install & bower install') + ' ...' +
```

Comment:
if something fails during install we still need to log info for user on how to do it manually